index.html

```diff
@@ -126,7 +127,35 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Laravel</a>
+                        <div class="cd-faq-content">
+                                <p>Eloquent (dostęp do bazy danych):</p>
+                                <ol>
+                                        <li>Dostępne są dwa query buildery: użycie <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php">pierwszego z nich</a> powoduje zwrócenie obiektów Active Record, natomiast <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Query/Builder.php">drugi</a> zwraca tablice.</li>
```

Comment:
To na pewno jest wada? Doctrine też pozwala wykonać query żeby zwróciło tablicę lub obiekty. Fakt, to powinna być opcja i hydrator a nie builder osobny ale samo istnienie dwóch builderów nie wydaje mi się jakoś tragiczne.